R/use_lintr.R

```diff
@@ -43,5 +43,37 @@ use_lintr <- function(path = ".", type = c("tidyverse", "full")) {
     )
   )
   write.dcf(the_config, config_file, width = Inf)
+
+  if (file.exists(file.path(path, "DESCRIPTION"))) {
+    # Some OS can only normalize a path if the associated file or folder exists, so the path needs to be re-normalized
+    tryCatch({
```

Comment:
I don't understand this part.
What do these systems return with `normalizePath("foo", mustWork = FALSE)` if foo doesn't exist?